src/main/java/io/xpire/logic/commands/DeleteCommand.java

```diff
@@ -3,31 +3,58 @@
 import static java.util.Objects.requireNonNull;
 
 import java.util.List;
+import java.util.Set;
+import java.util.TreeSet;
 
 import io.xpire.commons.core.Messages;
 import io.xpire.commons.core.index.Index;
 import io.xpire.logic.commands.exceptions.CommandException;
 import io.xpire.model.Model;
 import io.xpire.model.item.Item;
+import io.xpire.model.tag.Tag;
+import io.xpire.model.tag.TagComparator;
 
 /**
- * Deletes an item identified with its displayed index.
+ * Deletes an item identified with its displayed index or tag(s) associated with the item.
```

Comment:
if delete can not only delete items but also quality, should i also enable deleting reminder thru the command delete?

Reply:
Hmm... Would we need to delete reminders for items? Can we just set it back to 0 as the default